react/javascript/tile-visualization/src/components/DashboardTile/DashboardTile.js

```diff
@@ -29,6 +29,63 @@ import { ExtensionContext40 } from '@looker/extension-sdk-react'
 import { DashboardRunState } from '@looker/extension-sdk'
 import { LiquidFillGaugeViz } from '../LiquidFillGaugeViz/LiquidFillGaugeViz'
 
+const findTheLookModel = async (coreSDK) => {
+  try {
+    const models = await coreSDK.ok(
+      coreSDK.all_lookml_models({ fields: 'name' })
+    )
+    if (models.find((model) => model.name === 'cypress_thelook')) {
+      return { modelName: 'cypress_thelook' }
+    }
+    if (models.find((model) => model.name === 'thelook')) {
+      return { modelName: 'thelook' }
+    }
+    return {
+      error: 'No thelook model was found. Using a random value.',
+    }
+  } catch (error) {
+    console.error(error)
+    return {
+      error:
+        "An error occurred while reading the system's LookML models. Using a random value.",
+    }
+  }
+}
+
+const runInlineQuery = async (coreSDK, modelName) => {
+  try {
+    const response = await coreSDK.ok(
+      coreSDK.run_inline_query({
+        body: {
+          fields: ['users.count_percent_of_total', 'users.state'],
```

Comment:
Is `count_percent_of_total` a default measure in the cypress models? Looking at my own local copy of thelook I don't have it, but perhaps I just used the auto-generated LookML.

Reply:
It is. it's also there if you start services with content